File: Vitrola.Desktop/Models/Song.cs
using System;
namespace Vitrola.Desktop.Models
{
	public class Song
	{
		public string Id { get; set; } = string.Empty;
		public string SongId { get; set; } = string.Empty;
		public string Name { get; set; }
		public string SongUrl { get; set; }
		public string ThumbnailUrl { get; set; }
    }
}

File: Vitrola.Mobile/Vitrola.Mobile/Models/Song.swift
//
//  Song.swift
//  Vitrola.Mobile
//
//  Created by Natanael Ureña Fernández  on 12/3/23.
//

import Foundation

struct Song: Codable, Identifiable {
    var id: String
    var songId: String
    var name: String
    var songUrl: String
    var thumbnailUrl: String
}
